### DISCUSSION TITLE
[FINALIZADA] FrontendBR + Front in Sampa ~ VALENDO 1 INGRESSO ~

### DISCUSSION THREAD

**Discussion by diogomoretti** [frontendbr/forum#1117] [Dicas]
Faaaaaaala galera,

Seguinte, a parceria entre o **FrontendBR** e o [**Front in Sampa**](http://www.frontinsampa.com.br) continua a mil. Dessa vez vamos fazer uma brincadeira legal valendo **01 (um) ingresso para o evento** 🎉

# Como será?

### O FrontendBR fará uma surpresa no Front in Sampa (lá no hotel, no dia do evento). Queremos saber se vocês advinham o que será... Respondam abaixo =]

# 🔥 Divulgação da novidade e do vencedor(a)
!! UPDATE !!
- Vamos fechar a issue **segunda-feira, dia 25/Junho às 12h** 
- Vamos divulgar a novidade e ganhador(a) na **segunda-feira, dia 25/Junho às 14h** 

# 📋 Regras
- A primeira pessoa que acertar ganha. A segunda que acertar na ordem cronológica fica em 2º e assim vai.
- Se você por acaso ganhar, vamos entrar em contato para saber se realmente irá no evento. Caso negativo, passaremos para a próxima pessoa que acertou (respeitando a ordem cronológica)
- Mesmo que alguém acerte, não vamos comentar nada até a data de divulgação da novidade ;)
- Caso ninguém acerte "na mosca", vamos eleger as respostas que chegaram mais perto.

# ❓FAQ
- Posso responder mais de uma vez?
➖ _Sim_
- Não vou no evento ou já tenho ingresso, posso responder?
➖ _Sim_
- Eu sei o que é e vou no evento, posso responder?
➖ _Aconselhamos que deixe a galera tentar adivinhar para ser mais divertido e justo =]_

# ❤️ Agradecimento
Queríamos agradecer ao Fábio Vedovelli (@vedovelli). Final do mês ele vai se mudar e gentilmente doou o seu ingresso para o FrontendBR sortear. Obrigado Vedovelli e boa sorte nessa nova vida =]

#### BOA SORTE GALERA!

**Comment by ingridmazoni01** [comment]
acesso gratuito ao curso de React do Daciuk na Udemy

**Comment by eubond** [comment]
FrontendBR vai ter um programa. Vídeo, audio, quadrinhos.

**Comment by eubond** [comment]
Cursos FrontendBr

**Comment by Eduruiz** [comment]
acesso aos cursos de JS/React/Git do Daciuk na Udemy!

**Comment by espudaro** [comment]
Camiseta do FrontInSampa 2018

**Comment by espudaro** [comment]
Alguma palestra surpresa, que não está na grade do evento

**Comment by espudaro** [comment]
happy hour na faixa! \o/

**Comment by ninetails** [comment]
Podcast ao vivo '-'~

**Comment by anabastos** [comment]
Passar o jogo da copa

**Comment by felipedotcom** [comment]
Adesivo do frontendbr

**Comment by Eduruiz** [comment]
comida?! :heart:

**Comment by jezzipan** [comment]
A presença dos Racionais MC's.

**Comment by eroSilva** [comment]
FrontendBR vai ter um canal no youtube! >.<

**Comment by eubond** [comment]
FrontendBR na Copa da Rússia.

**Comment by eubond** [comment]
Frontendbr Reality Show

**Comment by eubond** [comment]
Anúncio de Meetup FrontendBR

**Comment by eubond** [comment]
Sorteio de Livro

**Comment by eroSilva** [comment]
Camisa do FrontendBR ^^,

**Comment by marioleme** [comment]
Camiseta do FrontInSampa 2018

**Comment by marioleme** [comment]
presença do  Maurício Samy Silva

**Comment by doug2k1** [comment]
FrontendBR vai migrar pro GitLab.

**Comment by rhake** [comment]
Migração GitLab

**Comment by felipedotcom** [comment]
Hackaton frontend-br

**Comment by ninetails** [comment]
Show ao vivo :laughing: 

Happy Hour substituído por uma churrascada :laughing:  :laughing:  :laughing:

**Comment by adrianoavelino** [comment]
Palestra com os fundadores

**Comment by adrianoavelino** [comment]
Meetup do frontendbr

**Comment by dougaraujos** [comment]
Agora para entrar tem que ser Fullstack

**Comment by felipedotcom** [comment]
Lives na twitch

**Comment by tobiassrocha** [comment]
Palestra surpresa.

**Comment by brunorafael8** [comment]
A surpresa vai ser um novo evento que vão criar

**Comment by adrianoavelino** [comment]
camiseta do frontendbr

**Comment by NandoSangenetto** [comment]
Code in the Dark

**Comment by NandoSangenetto** [comment]
Atração internacional

**Comment by adrianoavelino** [comment]
lançamento podcast frontendbr

**Comment by NandoSangenetto** [comment]
Transmissão ao vivo

**Comment by adrianoavelino** [comment]
lançamento de um produto oficial

**Comment by marioleme** [comment]
Transmissão online

**Comment by marioleme** [comment]
União com  o Google..

**Comment by adrianoavelino** [comment]
Divulgação do evento frontendbr

**Comment by ghost** [comment]
Sorteio de ingresso para a próxima edição.

**Comment by marioleme** [comment]
Participação da  Udemy com FrontSampa

**Comment by marcelgsantos** [comment]
Palestra do @elvisdetona! ;)

**Comment by marioleme** [comment]
@elvisdetona cantar  para todos...

**Comment by marioleme** [comment]
Sorteio de um  Computador!!!

**Comment by marioleme** [comment]
Palestra do Brendan Eich

**Comment by elvisdetona** [comment]
Só observando as respostas ahahahahah @marcelgsantos @marioleme

**Comment by marquinhusgoncalves-zz** [comment]
Bernard aparecendo de surpresa com uma nova fantasia !!!!

**Comment by guilhrmefp** [comment]
acesso a carreira frontend do alura

**Comment by felipefialho** [comment]
Olha ai @bernarddeluna 😄

**Comment by bernarddeluna** [comment]
:X

**Comment by fabih-ragnar-amon** [comment]
acho que é plataforma virtual para aprendizado front end!!! a preço acessível, claro! ♥

**Comment by fabih-ragnar-amon** [comment]
cursos para quem não sabe nada e quer se tornar front fodão!! hehehe

**Comment by praganavictor** [comment]
Vai ter um mascote no evento :smile:

**Comment by praganavictor** [comment]
Palestra surpresa com Zeno?

**Comment by felipefialho** [comment]
E ai @zenorocha?

**Comment by felipedotcom** [comment]
Mentoria

**Comment by felipedotcom** [comment]
Curso de vuejs

**Comment by fabih-ragnar-amon** [comment]
ceis vão virar ONG e vão chamar só a mulherada que não sabe nada pra aprender?!?! (gente, to tentando ganhar esse ingresso!! ♥)

**Comment by espudaro** [comment]
Vai ter uma sala "vip", estilo da Globo no carnaval, que basicamente será uma live, onde os profissionais poderão trocar ideias ou comentar sobre as palestras. Seria massa!!

**Comment by bernarddeluna** [comment]
@fargoud olá :) tudo bem? quem é você? Tem linkedin ou twitter? Talvez possa ajudar.

**Comment by edias15** [comment]
Sorteio de 1 final de semana no Hotel.

**Comment by tonimota** [comment]
Lançamento de Camisetas "frontendbr"

**Comment by tonimota** [comment]
Divulgação de um canal de Podcasts feito por vocês.

**Comment by praganavictor** [comment]
Empresas buscando profissionais de frontend?

**Comment by praganavictor** [comment]
Coisas gratis?

**Comment by alessandragiacon** [comment]
Canal no IGTV

**Comment by gss-patricia** [comment]
Frontendbr vai virar uma startup.

**Comment by gss-patricia** [comment]
Frontendbr vai fazer uma abertura épica com assunto nerd

**Comment by Raffesmind** [comment]
Vai ter uma transmissão online para todos que não estiverem ao vivo assistirem

**Comment by gss-patricia** [comment]
Frontendbr on the road!

**Comment by gss-patricia** [comment]
Frontendbr vai fazer alguma dinâmica, desafio ou brincadeira para os participantes que estarão na plateia. Tipo o Frontin Campinas que fez uma versão do imagem e ação

**Comment by fabih-ragnar-amon** [comment]
@bernarddeluna tenho twitter sim! @FabihArgoud

**Comment by romulobastos** [comment]
Lançamento do Canal de vídeos FrontendBr!

**Comment by diogomoretti** [comment]
Issue fechada. Daqui a pouco vamos divulgar a novidade e o ganhador!

**Comment by diogomoretti** [comment]
Depois de algumas análises em todos os comentários, vimos que o comentário do @espudaro foi o que chegou mais perto (aliás, é bem dizer o que ele falou). **Portanto ele é o ganhador do ingresso**.

Já entramos em contato com ele e passamos o ingresso.

Aguarde uma nova issue aqui contando da novidade e também nas redes sociais do FrontendBR =]

Mais uma vez, OBRIGADO A TODOS PELA PARTICIPAÇÃO NA BRINCADEIRA. Dia 14 tem mais. Até lá pessoal =]
